### ISSUE TITLE
Git-lfs installation failed with exit code: 1

### ISSUE THREAD

**Issue by mandalorianbob** [microsoft/azure-pipelines-agent#3343]
## Agent Version and Platform
Version of your agent? 2.183.1

OS of the machine running the agent? Windows

## Azure DevOps Type and Version
dev.azure.com (formerly visualstudio.com)

If dev.azure.com, what is your organization name? microsoftcasualgames

## What's not working?
Prepending Path environment variable with directory containing 'git.exe'.
git version
git version 2.28.0.windows.1
git init "D:\a\_work\1\s"
Initialized empty Git repository in D:/a/_work/1/s/.git/
git remote add origin [GIT_REPO]
git config gc.auto 0
git config --get-all [GIT_REPO].extraheader
git config --get-all http.proxy
git config http.version HTTP/1.1
git lfs install --local
git: 'lfs' is not a git command. See 'git --help'.
##[error]Git-lfs installation failed with exit code: 1

I've run into this before, and the only solution was to install LFS manually on the agent. This is on a managed image, so I'm not really able to access the machine to install LFS manually, so that's not a solution now.

Is there something I can do to fix this?

**Comment by anatolybolshakov** [comment]
Hi @mandalorianbob do you use self-hosted agent or ms hosted one?

**Comment by mandalorianbob** [comment]
I guess technically it would be "self-hosted", but it's a 1ES Managed Image. So I don't really have access to it, as though it were MS Hosted. But I'm not using the hosted pools.

**Comment by anatolybolshakov** [comment]
@mandalorianbob to use git lfs - it should be already installed on agent environment, or you need access to install it on this environment.

**Comment by mandalorianbob** [comment]
I agree that it should already be installed haha. Do you have any best practices for how to install git lfs to the agent then?  I can run a powershell script that'll download it from the website and install it I guess.

**Comment by mandalorianbob** [comment]
If I wanted to manually set the path to point to git lfs, where would it be, relative to the agent folder? Just installed in program files or what?
